### PR TITLE
Activated the pointer cursor and center the color panel for the color picker

### PR DESCRIPTION
**Pull request by XavierCLL**
Some minor fixes for the color picker:

Before:
image

After:
image

### CODE REVIEW

**Comment by DanWismer** [comment]
Thanks @XavierCLL , this is great!

We have configured our branches now where new features need to branch off the `dev` branch. Once we test the feature locally and all is good,  we `squash and merge` features into `dev`. Then we can see the changes live on our dev server (dev server is listening to our dev branch). Once we see things working fine on the dev server, we then do a dev `merge commit` into `main`.

In short, we ask that all new features branch off of dev instead of main. And then for PR would be to merge the feature into dev. Does that make sense? Is this something you can change?

Thanks for your contribution.
